GitCommands/Settings/AppSettings.cs

```diff
@@ -637,7 +637,7 @@ public static bool CloseCommitDialogAfterLastCommit
             set => SetBool("closecommitdialogafterlastcommit", value);
         }
 
-        public static bool RefreshCommitDialogOnFormFocus
+        public static bool RefreshArtificialCommitOnApplicationActivated
         {
             get => GetBool("refreshcommitdialogonformfocus", false);
             set => SetBool("refreshcommitdialogonformfocus", value);
```

Comment:
Should I also adapt the name "refreshcommitdialogonformfocus" in the settings file?

How to migrate the old `AppSettings`?
`get => GetBool("RefreshArtificialCommitOnApplicationActivated", GetBool("refreshcommitdialogonformfocus", false));`

Reply:
Do not bother...